metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/MarcXmlEncoder.java

```diff
@@ -247,11 +239,12 @@ private static class Encoder extends DefaultStreamPipe<ObjectReceiver<String>> {
         private String currentEntity = "";
 
         private boolean emitNamespace = true;
-        private Object[] namespacePrefix = new Object[]{emitNamespace ? NAMESPACE_PREFIX : EMPTY};
+        private Object[] namespacePrefix = new Object[]{NAMESPACE_PREFIX};
```

Comment:
Why did you change this? Now one has to remember to adjust this line as well should the default for `emitNamespace` be switched one day.